overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -94,6 +94,15 @@ func addCommand(name, shortHelp, longHelp string, generator func() command) *com
 	return cmd
 }
 
+// UnsuccessfulError carries a specific exit code to be returned to the client.
+type UnsuccessfulError struct {
+	ExitCode int
+}
+
+func (e UnsuccessfulError) Error() string {
+	return fmt.Sprintf("error code: %d", e.ExitCode)
```

Comment:
"unsuccessful with exit code: %d"

Reply:
Will do. FWTW it's not expected to surface like this with current code, is just to satisfy error api.

Reply:
I know, but this makes it a bit more greppable if it does surface at some point

Reply:
Updated.